layer/unittest/trace_event_log_tests.cc

```diff
@@ -393,5 +423,62 @@ TEST(TraceEventDeathTest, InvalidCompleteEvent) {
 }
 #endif
 
+TEST(TraceEventLogger, MethodCheck) {
+  StringOutput out = {};
+  TraceEventLogger logger(&out);
+  InstantEvent instant_event("compile_time_init", "compile_time", 123, 321);
+
+  logger.StartLog();
+  logger.AddEvent(&instant_event);
+  logger.Flush();
+  logger.EndLog();
+  // Checks double `EndLog` calls.
+  logger.EndLog();
+  std::ostringstream expected_str;
+  expected_str
+      << R"({ "name" : "compile_time_init", "ph" : "i", "cat" : "compile_time", "pid" : 123, "tid" : 321, "ts" : )"
+      << std::fixed << ToUnixMillis(instant_event.GetCreationTime().GetValue())
```

Comment:
Can we pass some fixed creation time when we create the event or just after? I think this would make these tests a little bit easier -- we could embed the exact time  in the string